Lib/asyncio/tasks.py

```diff
@@ -25,6 +25,9 @@
 from . import futures
 from .coroutines import _is_coroutine
 
+_SENTINEL = enum.Enum("_SENTINEL", "sentinel")
```

Comment:
Also, is there any risk with the value being an int? I initially found this solution from stackoverflow, where it defines the enum value as `object()`: https://stackoverflow.com/a/60605919